pkg/cmd/env.go

```diff
@@ -59,7 +59,7 @@ func NewEnvInitCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Command
 	var envArgs types.EnvMeta
 	ctx := context.Background()
 	cmd := &cobra.Command{
-		Use:                   "env:init",
+		Use:                   "env:init <envName>",
```

Comment:
If `--namespace` is not set, `default` namespace will be created for the env?

Reply:
yes